### PR TITLE
[MAP] Several major remaps, and additions #3

### PR DESCRIPTION
**Pull request by Juani2400**
Mapwise, these are the changes:
- Remap: Atmopsherics.
- Remap: Captain's Office.
- Remap: Head of Staff Conference Room.
- Remap: Chief Medical Officer's Office.
- Remap: Medical Briefing Room.
- Remap: Security Processing and Holding Cell.
- Modification: Robotics Laboratory.
- Addition: Desk Ringers in several locations.
- Fix: #3391, #3374, #3345, #3341, #3308, #3306, #3277, #3244, #3173,
#3094.
- Several other minor changes.

Codewise, the following has been modified:
- A new subtype of folder has been sprited for the Security folders.
- More colours of flooring decals were added.
- New colours for pipes were added (Lohikar).

Images:

image

image

image

image

image

### CODE REVIEW

**Comment by Lohikar** [comment]
CentComm is a DMM, not TGM. Must be fixed before this is mergable.

File: code/game/turfs/flooring/flooring_decals.dm [hunk 1]
```diff
@@ -15,6 +15,7 @@
 		var/cache_key = "[alpha]-[color]-[dir]-[icon_state]-[layer]"
 		if(!floor_decals[cache_key])
 			var/image/I = image(icon = src.icon, icon_state = src.icon_state, dir = src.dir)
+			I.layer = T.layer
```
Comment (Lohikar) [inline code/game/turfs/flooring/flooring_decals.dm]: !

Comment (Juani2400) [reply code/game/turfs/flooring/flooring_decals.dm]: I never touched that.

**Comment by skull132** [comment]
Changelog required.

**Comment by Lohikar** [comment]
Bumping this to high priority because I need to make a map change and it'll likely conflict.

**Comment by Juani2400** [comment]
The changelog and the requested changes are done.

**Pull request merged**